src/evaluate.py

```diff
@@ -163,12 +166,13 @@ def write_preds(tasks: Iterable[tasks_module.Task], all_preds, pred_dir, split_n
                  'mrpc': 'MRPC',
                  'qnli': 'QNLI',
                  'qqp': 'QQP',
-                 'rte': 'RTE',
+                 'rte-glue': 'RTE',
```

Comment:
It seems like it'd be more robust to keep the 'rte' object name for the GLUE version and use super-rte for the SuperGLUE version. No

Reply:
Cut off? I'm fine either way.